arcon_state/src/backend/metrics_utils.rs

```diff
@@ -0,0 +1,14 @@
+#[cfg(feature = "metrics")]
```

Comment:
You can remove all the cfg feature lines here in this file. 
You just need to have it covered under the flag when declaring the module.

```rust
// src/backend/mod.rs
#[cfg(feature = "metrics")]
pub mod metrics_utils;
```